src/app/models/reservation.js

```diff
@@ -0,0 +1,23 @@
+import Backbone from 'backbone';
+
+const Reservation = Backbone.Model.extend({
+
+  validate(attributes) {
+    const errors = {};
+
+    if (!attributes.name) {
+      errors.name = ['cannot be blank'];
+    }
+
+    if (!attributes.email) {
+      errors.email = ['cannot be blank'];
+    }
+  },
+
+  urlRoot: function () {
+   return 'https://ada-backtrek-api.herokuapp.com/trips/' + this.attributes.tripId + '/reservations';
```

Comment:
It's preferable to use interpolation instead of concatenation (just a style criticism).

like:
```
`   return `https://ada-backtrek-api.herokuapp.com/trips/${this.attributes.tripId}/reservations`;
```